<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>535</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Disconnected</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="handleWidth">
       <number>4</number>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <layout class="QVBoxLayout" name="layout_channels" stretch="0,0">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QListWidget" name="listview_channels">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="cInput"/>
          </item>
          <item>
           <widget class="QPushButton" name="cJoin">
            <property name="text">
             <string>Join</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QWebView" name="baseChatWindow">
          <property name="url">
           <url>
            <string>about:blank</string>
           </url>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="wChannel">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="wInput"/>
          </item>
          <item>
           <widget class="QPushButton" name="wSend">
            <property name="text">
             <string>Send</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>734</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPenis">
    <property name="title">
     <string>Penis</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEmote_Menu">
    <property name="title">
     <string>Emotes</string>
    </property>
    <widget class="QMenu" name="menuBTTV_Emotes">
     <property name="title">
      <string>BTTV Emotes</string>
     </property>
     <addaction name="actionBTTV_Global_Emotes"/>
     <addaction name="actionBTTV_Channel_Emotes"/>
     <addaction name="actionBTTV_GIF_Emotes_2"/>
    </widget>
    <widget class="QMenu" name="menuFFZ_Emotes_2">
     <property name="title">
      <string>FFZ Emotes</string>
     </property>
     <addaction name="actionFFZ_Global_Emotes_2"/>
     <addaction name="actionFFZ_Channel_Emotes_2"/>
    </widget>
    <addaction name="actionDisable_all_emotes"/>
    <addaction name="menuBTTV_Emotes"/>
    <addaction name="menuFFZ_Emotes_2"/>
   </widget>
   <addaction name="menuPenis"/>
   <addaction name="menuEmote_Menu"/>
  </widget>
  <action name="actionEnable_BTTV_Emotes">
   <property name="text">
    <string>Enable BTTV Emotes</string>
   </property>
  </action>
  <action name="actionDisable_all_emotes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Twitch Emotes</string>
   </property>
  </action>
  <action name="actionBTTV_GIF_Emotes">
   <property name="text">
    <string>BTTV GIF Emotes</string>
   </property>
  </action>
  <action name="actionBTTV_Global_Emotes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>BTTV Global Emotes</string>
   </property>
  </action>
  <action name="actionBTTV_Channel_Emotes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>BTTV Channel Emotes</string>
   </property>
  </action>
  <action name="actionBTTV_GIF_Emotes_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>BTTV GIF Emotes</string>
   </property>
  </action>
  <action name="actionFFZ_Global_Emotes">
   <property name="text">
    <string>FFZ Global Emotes</string>
   </property>
  </action>
  <action name="actionFFZ_Channel_Emotes">
   <property name="text">
    <string>FFZ Channel Emotes</string>
   </property>
  </action>
  <action name="actionFFZ_Global_Emotes_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>FFZ Global Emotes</string>
   </property>
  </action>
  <action name="actionFFZ_Channel_Emotes_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>FFZ Channel Emotes</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cJoin</tabstop>
  <tabstop>wSend</tabstop>
  <tabstop>cInput</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
